components/raftstore/src/store/snap.rs

```diff
@@ -123,6 +124,8 @@ pub struct SnapKey {
     pub region_id: u64,
     pub term: u64,
     pub idx: u64,
+    pub start: u64,
```

Comment:
It seems that we should not add these two member here. Then we can not get the expected snapshot by `(region_id, term, idx)` from registry.